Markdown/Markdown.sublime-syntax

```diff
@@ -643,13 +643,18 @@ contexts:
     - include: inline-bold-italic-linebreak
     - include: scope:text.html.basic
   raw:
-    - match: '(`+)'
-      scope: punctuation.definition.raw.begin.markdown
-      push:
-        - meta_scope: markup.raw.inline.markdown
-        - match: '\1'
-          scope: punctuation.definition.raw.end.markdown
-          pop: true
+    - match: '{{backticks}}'
```

Comment:
I suggest this instead:

```yaml
  raw:
    - match: '(`+)'
      scope: punctuation.definition.raw.begin.markdown
      push:
        - meta_scope: markup.raw.inline.markdown
        - match: '\1'
          scope: punctuation.definition.raw.end.markdown
          pop: true
        - include: raw
```


Should probably also exit on empty lines. See:

test `test
 
test`